ContributedExamples/plot_mht.ipynb

```diff
@@ -4,27 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>If you look at the output of <code>mht.time</code>, you can see that the <code>n=3</code> argument in this case chooses 6 years of output, not one.</p><p> </p><p>Suggest explaining that the unit conversion is from W to PW.</p><p> </p><p>Is there a reason to output the shape of the array?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I will update the number of years. I did not check this, and thanks for including the conversion unit. About checking the array shape, I have no idea. It was in the original notebook.